src/markdown-pages/install-with-kurl/reducing-storage.md

```diff
@@ -0,0 +1,211 @@
+---
+path: "/docs/install-with-kurl/changing-storage"
+date: "2022-10-10"
+weight: 27
+linktitle: "Changing Storage"
+title: "Changing Storage"
+isAlpha: false
+---
+
+It is relatively common to initially allocate less (or more) storage than is required for an installation in practice.
+Some storage providers allow this to be done easily, while others require far more effort.
+
+
+# OpenEBS LocalPV
+
+OpenEBS LocalPV consumes a host's storage at `/var/openebs/local`.
+If you increase the amount of storage available here, OpenEBS will use it.
+If it shrinks, OpenEBS will have less storage to use.
+
+# Rook-Ceph
+
+## Expanding Storage
+
+Rook does not support expanding the storage of existing block devices, only adding new ones.
+However, a 100GB block device added to a node that already had a 100GB block device used by rook will be treated similarly to a freshly created instance with a single 200GB disk.
```

Comment:
I'm not sure if we want to go into detail here, this is about changing sizes but it's worth considering.

Rook assumes disk, or at least machines, have the same amount of storage available. If you for example add a 100GB disk to 1 node and not the other 2 you won't get any extra storage. If you add 100GB to one machine and 200GB to the other two, you'll get ~100GB of usable additional storage.

Maybe here we should just state that all machines should have the same amount of 'total storage' committed to Rook or the overall available storage will be less than the physical stroage.

Reply:
I think this depends on how many machines you have

If you have three machines, it's accurate - but if you have 5 or 1 I don't think it is

Reply:
Then should you explain that replication is set to 2 or 3 and how that affects expanding disks?